### PR TITLE
Proofreading of the skipped.md, conclusion.md and glossary.md chapters

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: book/embeds.md [hunk 1]
```diff
@@ -823,7 +823,7 @@ class Tab:
 ```
 
 Note that the guts of `load` now live in the `Frame`, because
-the `Frame` owns the DOM tree. The `Frame` can *also* construct child
+the `Frame` owns the HTML tree. The `Frame` can *also* construct child
```
Comment (chrishtr) [inline book/embeds.md:826]: HTML tree or DOM. Not DOM tree.

File: book/glossary.md [hunk 2]
```diff
@@ -150,11 +150,6 @@ Document
 : The conceptual object created when loading a web page and
 modified by interacting with it, an analogy to physical documents.
 
-Document tree
```
Comment (chrishtr) [inline book/glossary.md:153]: The book doesn't use this term, it uses HTML tree. So moved.

File: book/html.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Constructing a Document Tree
```
Comment (chrishtr) [inline book/html.md:2]: This is the only place in this chapter using that terminology. Changed to be consistent.

File: book/skipped.md [hunk 4]
```diff
@@ -194,7 +194,7 @@ they have batteries of hundreds of thousands of [unit] and [integration] tests.
 Recently, a lot of focus has been put on robust
 [cross-browser tests](https://wpt.fyi) that allow a single automated test
 to run on all browsers to verify that they all behave the same on the same
-input. And there are now yearly [Interoperability]^["Interop", for short.]
+input. And there are now yearly [interoperability]^["Interop", for short.]
```
Comment (chrishtr) [inline book/skipped.md:197]: Interop is a proper noun because it's the specific benchmark; interoperability is a concept.

File: book/glossary.md [hunk 5]
```diff
@@ -74,7 +74,7 @@ navigate web pages. Also often just called a "browser".
 
 Web page
 
-: The basic unit of the web; defined by unique URL.
+: The basic unit of the web; defined by unique URL that returns HTML.
```
Comment (pavpanchekha) [inline book/glossary.md:77]:
```suggestion
: The basic unit of the web; defined by a unique URL that returns HTML.
```

**Review by pavpanchekha — APPROVED** [review]

**Pull request merged**